lib/lbt/bundle/BundleDefinition.js

```diff
@@ -18,7 +18,13 @@ const SectionType = {
 	Preload: "preload",
 
 	/**
-	 * For each module, a jQuery.sap.require call will be created.
+	 * Content information for another bundle is written.
+	 * Requires UI5 version 1.74.0 which adds support for the 'bundles' configuration.
```

Comment:
maybe ... **runtime** support for the 'bundles' and 'bundlesUI5' ...

Ignore my comment if you meant the bundle configuration of the tooling. But then there would be no dependency to 1.74, so I guess you mean the runtime (or loader) configuration option.

Reply:
Done